pygmt/src/pygmtlogo.py

```diff
@@ -0,0 +1,266 @@
+
+    # -----------------------------------------------------------------------------
+    # Helpful definitions
+    # -----------------------------------------------------------------------------
+    size = 4
+    region = [-size, size] * 2
+    projection = "x1c"
+
+    # Outer and inner radii of compass lines
+    r1, r2 = size * 0.625, size * 0.325
+
+    # Rotation around z (vertical) axis placed in the center
+    # Has to be applied to each plotting command, up on second call set to True
+    perspective = "30+w0/0"  # Rotation by 30 degrees
+
+    # -----------------------------------------------------------------------------
+    # Define colors
+    # -----------------------------------------------------------------------------
+    color_light = "white"
+    color_dark = "gray20"
+
+    # visual
+    color_blue = "48/105/152"  # Python blue
+    color_yellow = "255/212/59"  # Python yellow
+    color_red = "238/86/52"  # GMT red
+    if not color:
+        color_blue = color_yellow = color_red = color_dark
+        if theme == "dark":
+            color_blue = color_yellow = color_red = color_light
+
+    # background and wordmark
+    color_bg = color_light
+    color_py = color_blue
+    color_gmt = color_dark
+    if theme == "dark":
+        color_bg = color_dark
+        color_py = color_yellow
+        color_gmt = color_light
+
+    # -----------------------------------------------------------------------------
+    # Define shape
+    # -----------------------------------------------------------------------------
+    symbol = "c"  # circle
+    diameter = 7.5
+    diameter_add = 0.5
+    if shape == "hexagon":
+        symbol = "h"  # hexagon
+        diameter = 8.6
+        diameter_add = 0.6
+
+    # -----------------------------------------------------------------------------
+    # Define wordmark
+    # -----------------------------------------------------------------------------
+    font = "AvantGarde-Book"
+    match wordmark:
+        case "vertical":
+            args_text_wm = {
+                "x": 0,
+                "y": -5,
+                "justify": "CT",
+                "font": f"2.5c,{font}",
+            }
+        case True | "horizontal":
+            args_text_wm = {
+                "x": 6,
+                "y": 0,
+                "justify": "LM",
+                "font": f"8c,{font}",
+            }
+
+    # -----------------------------------------------------------------------------
+    # Start plotting
+    # -----------------------------------------------------------------------------
+    fig = pygmt.Figure()
+
+    # .............................................................................
+    # blue circle / hexagon for Earth
+    # .............................................................................
+    fig.plot(
+        x=0,
+        y=0,
+        region=region,
+        projection=projection,
+        style=f"{symbol}{diameter}c",
+        pen=f"15p,{color_blue}",
```

Comment:
The pen thickness should be exactly 0.5c to avoid the tiny blue patch at the tail of the arrow 
<img width="830" alt="image" src="https://github.com/user-attachments/assets/b1a5e906-fd95-4819-acb7-241b21c7d493" />

```suggestion
        pen=f"0.5c,{color_blue}",
```

Reply:
Nice catch. Thanks!